bowling.js

```diff
@@ -0,0 +1,125 @@
+
+class Bowling{
+    constructor() {
+        this.score = 0;
+        this.frame_no = 1;
+        this.frame = [];
+        this.max_frame = 11;
+        this.spare = false
+        this.results = [];
+        this.extra_rolls = false;
+    };
+
+
+    newRoll(pins) {
+        this.results.push(pins);
+        if(this.extra_rolls === 1){
+            this.finalFrameClac();
```

Comment:
slight typo here 'finalFrameClac' it ended up being a little misleading as I didn't understand what it was at first, but managed to figure it out